homework.md

```diff
@@ -44,61 +44,63 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A scripting language used to create dynamic, interactive web pages
 
 ## Question 8
 
-What three problem does Git & Github solve?
+What three problems does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. It makes file sharing easy
+2. Tracking changes made to a file is simple
+3. It facilitates collaboration and team work
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+A carbon copy of the repository is stored in a local GitHub server
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+It is a request we make in order for changes made to a repository be accepted by its owner.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It is a record of detailed information about changes made to a file in a Git repository
 
 ## Question 13
 
-What does it meant to "push" changes to Github?
+What does it mean to "push" changes to GitHub?
 
-You send the changes of your local repository to Github
+You send your local repository to GitHub
 
 ## Question 14
 
-What is Github?
+What is GitHub?
 
-A place for us to store and share our code.
+A website where Git is utilized.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save the changes you made to a file in GitHub
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Microsoft Visual Studio Code
 
```

Comment:
Hi Michael, nice work! Would you look into adding another solution for question 18?

Reply:
Hi Lisha,
Thanks for the feedback. I'll check that one.